src/Holding/CreateHolding/CreateHolding.test.js

```diff
@@ -0,0 +1,80 @@
+import '../../../test/jest/__mock__';
```

Comment:
This line is redundant. Could you remove it?
All set up of mocks import is implemented here - https://github.com/folio-org/ui-inventory/blob/master/jest.config.js